src/cli/CommandLineIface.cpp

```diff
@@ -245,3 +254,48 @@ void CommandLineIface::outputTranslation(Translation output) {
     outstream_.flush();
     eventLoop_.exit(); // Unblock the main thread
 }
+
+int CommandLineIface::allowNativeMessagingClient(QStringList ids) {
+    if (ids.isEmpty()) {
+        qCritical().noquote() << "No client ids specified";
+        return 1;
+    }
+
+    auto clients = settings_.nativeMessagingClients();
+#if (QT_VERSION < QT_VERSION_CHECK(5, 14, 0))
+    clients += ids.toSet();
+#else
+    clients += QSet<QString>(ids.begin(), ids.end());
+#endif
+    settings_.nativeMessagingClients.setValue(clients);
+    return updateNativeMessagingManifests();
+}
+
+int CommandLineIface::removeNativeMessagingClient(QStringList ids) {
+    if (ids.isEmpty()) {
+        qCritical().noquote() << "No client ids specified";
+        return 1;
+    }
+
+    auto clients = settings_.nativeMessagingClients();
+#if (QT_VERSION < QT_VERSION_CHECK(5, 14, 0))
+    clients += ids.toSet();
```

Comment:
This seems to be wrong. You probably want `-=` here. I think you may have accidentally copy pasted it?

Reply:
Yes! Nice catch! Thanks!